models/account.model.js

```diff
@@ -12,6 +12,10 @@ const AccountSchema = new mongoose.Schema({
         type: String,
         required: true,
     },
+    pronoun: {
+        type: String,
+        required: true,
```

Comment:
Should pronoun specification be optional?

Reply:
I had a commit to do this sitting in my GitHub not pushed. It's up now.